src/pages/gen2/deploy-and-host/sandbox-environments/features/index.mdx

```diff
@@ -59,53 +98,57 @@ export const auth = defineAuth({
 });
 ```
 
-### Retrieve secrets
-
-To get the value of a secret from the cloud, run the following command in your terminal:
-
-```bash
-npx amplify sandbox secret get foo
- name: foo
- version: 1
- value: 123
- lastUpdated: Mon Nov 13 2023 22:19:12 GMT-0800 (Pacific Standard Time)
-```
-
-### Remove secrets
+The `secret()` function does NOT retrieve the value of the secret. It places a reference to the secret value in the backend definition. The secret value is only resolved during deployment of your backend.
 
-To remove a secret from the cloud, run the following command in your terminal:
+The `secret()` function can only be used in specific places in your backend definition such as [configuring auth providers](/gen2/build-a-backend/auth/add-social-provider/#configure-social-sign-in-backend) and [granting function secret access](/gen2/build-a-backend/functions/#secret-access).
 
-```bash
-npx amplify sandbox secret remove foo
-```
+<Callout info>
+  To deploy a backend that uses `secret()` references via Amplify hosting, the
+  secret values must be [configured for the Amplify app or
+  branch](/gen2/deploy-and-host/fullstack-branching/secrets-and-vars)
+</Callout>
 
-### Work with multiple AWS profiles
+## Work with multiple AWS profiles
 
-Sometimes you might have multiple AWS profiles set up locally. To run `amplify sandbox secret` commands, use the `--profile` flag to deploy to a specific profile. For example, let's say you have two AWS profiles set up locally—`default` and `work`. To add secrets to the `work` profile, run the following command in your terminal:
+Sometimes you might have multiple AWS profiles set up locally. To run `amplify sandbox secret` commands, use the `--profile` flag to deploy to a specific profile. For example, let's say you have two AWS profiles set up locally—`default` and `work`. To add secrets to the sandbox in the `work` profile, run the following command in your terminal:
 
 ```bash
 npx amplify sandbox secret set foo --profile work
 ```
 
-## Multiple sandboxes per app
+## Work with multiple named sandboxes
 
 <Callout info>
-  Provisioning multiple sandboxes per app is possible but not recommended because managing multiple ephemeral environments for a single app introduces complexity. With multiple sandboxes, it can be difficult to keep track of what code version or configuration is deployed where. Sticking to a single ephemeral sandbox per app keeps your workflows simple and straightforward.
+  Provisioning multiple sandboxes per app is possible but not recommended
+  because managing multiple ephemeral environments for a single developer
+  introduces complexity. With multiple sandboxes, it can be difficult to keep
+  track of what code version or configuration is deployed where. Sticking to a
+  single sandbox per developer keeps your workflows simpler.
 </Callout>
```

Comment:
```suggestion

Provisioning multiple sandboxes per app is possible but not recommended because managing multiple ephemeral environments for a single developer introduces complexity. With multiple sandboxes, it can be difficult to keep track of what code version or configuration is deployed where. Sticking to a single sandbox per developer keeps your workflows simpler.

</Callout>
```